src/AppBundle/Entity/Post.php

```diff
@@ -80,10 +82,21 @@ class Post
      */
     private $comments;
 
+    /**
+     * @var Collection|Tag[]
```

Comment:
@xabbuh, but collections support many useful methods: `each`, `map`, `filter` and `partition`